spec/lib/importers/article_importer_spec.rb

```diff
@@ -67,5 +68,50 @@
         expect(Article.find_by(title: 'Istanbul').mw_page_id).to eq(3391396)
       end
     end
+
+    it 'returns the correct number of titles without raising error 414' do
```

Comment:
This looks like a great test. It should include a comment explaining in a little more detail what the test is doing and why the list of titles is what it is.